src/codec_dav1d.c

```diff
@@ -54,7 +54,26 @@ static avifBool dav1dCodecOpen(avifCodec * codec, avifDecoder * decoder)
     if (codec->internal->dav1dContext == NULL) {
         // Give all available threads to decode a single frame as fast as possible
         codec->internal->dav1dSettings.n_frame_threads = 1;
-        codec->internal->dav1dSettings.n_tile_threads = AVIF_CLAMP(decoder->maxThreads, 1, DAV1D_MAX_TILE_THREADS);
+        int maxThreads = AVIF_MAX(decoder->maxThreads, 1);
+        int tileThreads = AVIF_MIN(maxThreads, DAV1D_MAX_TILE_THREADS);
+#ifdef DAV1D_MAX_POSTFILTER_THREADS
+        // In dav1d 0.8.2 or later (dav1d API version 5.0.1 or later), n_postfilter_threads is
+        // available. The tile threads are way more useful than the postfilter threads, so allocate
+        // a quarter or so of the threads to postfilters and the rest to tiles.
+        int postfilterThreads;
+        if (maxThreads <= 8) {
+            postfilterThreads = (maxThreads + 3) / 4; // ceil(maxThreads / 4.0)
+        } else {
+            postfilterThreads = AVIF_MIN(maxThreads / 4, DAV1D_MAX_POSTFILTER_THREADS);
+        }
+        // If n_postfilter_threads is equal to 1, dav1d won't create any postfilter threads.
```

Comment:
well in fact even if the postfilters aren't parallelized they're still running in their own thread (the frame thread, or in your case the application thread), since all the entropy decoding / block recon is happening in the tile threads, that means they run linearly, but still in parallel of the tile threading stuff. So you effectively have n_tile_threads + n_pf_threads, in any case, even if the latter is one. (basically if any of these 2 values is 1, your application thread does actual work, so imo you should consider that as part of your maxThreads. Obviously tho if both are 1 you just have your app thread active, nothing is spawned).